packages/pubsub/test/peer-streams.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { generateKeyPair } from '@libp2p/crypto/keys'
+import { type PeerId } from '@libp2p/interface'
+import { defaultLogger } from '@libp2p/logger'
+import { peerIdFromPrivateKey } from '@libp2p/peer-id'
+import { expect } from 'aegir/chai'
+import * as lp from 'it-length-prefixed'
+import { pipe } from 'it-pipe'
+import { Uint8ArrayList } from 'uint8arraylist'
+import { PeerStreams } from '../src/peer-streams.js'
+import { ConnectionPair } from './utils/index.js'
+
+describe('PeerStreams large message handling', () => {
+  let otherPeerId: PeerId
+
+  beforeEach(async () => {
+    otherPeerId = peerIdFromPrivateKey(await generateKeyPair('Ed25519'))
+  })
+
+  it('should receive messages larger than MAX_DATA_LENGTH when maxDataLength is set', async () => {
```

Comment:
```suggestion
  it('should receive messages larger than internal MAX_DATA_LENGTH when maxDataLength is set', async () => {
```

maybe i'm being pedantic here, but reading this was slightly confusing

Reply:
Yes you're right, only who knows the source code of `it-length-prefixed` would have understood that.